python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,7 +42,7 @@ def _getName(obj):
 
 _imported_GangaList = None
 
-do_not_copy = ['_index_cache', '_parent', '_registry', '_data', '_read_lock', '_write_lock', '_proxyObject']
+do_not_copy = ['_index_cache', '_index_cache_dict', '_parent', '_registry', '_data', '_data_dict', '_read_lock', '_write_lock', '_proxyObject']
```

Comment:
I just got put off when I printed it and it contained a string rep of the full contents of a huge list rather than just some unique name of the list I forget how this works behind the scenes but I think form of string rep of objects is taken during the copy process.
Back to the original point I think you're correct and can drop ```_index_cache``` from the list probably as well as ```_data``` but the tests will tell us if it's safe